scripts/build_ubuntu.sh

```diff
@@ -17,4 +17,4 @@ mkdir -p "$TEST_INSTALL_DESTDIR"
 LDFLAGS="-s" "$GITREPO_ROOT/scripts/test.sh"
 
 git -C "$GITREPO_ROOT" fetch --tags --unshallow || true
-git -C "$GITREPO_ROOT" describe --long > "$TEST_INSTALL_DESTDIR/aktualizr-version"
+"$GITREPO_ROOT/scripts/get_version.sh" > "$TEST_INSTALL_DESTDIR/aktualizr-version"
```

Comment:
Do you still need to provide the git repo path here for this to work?

Reply:
It's computed from the scripts' directory on line 6 if left undefined.

Reply:
It's computed from the script's current directory if left undefined, on line 6.

Reply:
Ok I see. I've added a fix for that.

Also made `make_src_archive.sh` smarter.